Generate/src/Medication-9-0-7/XIS-Server/DispenseRequest/mp9-vv-fhir3-0-2-xis-10-1.xml

```diff
@@ -11,19 +11,16 @@
       <name value="Scenario 10.1"/>
       <description value="Scenario 10.1 - HPK geneesmiddel"/>
       <nts:include value="xis-ma-vv-operation-search">
-         <nts:with-parameter name="description"
-                       value="Test XIS server to serve DispenseRequest (NL: VerstrekkingsVerzoek) - HPK geneesmiddel"/>
+         <nts:with-parameter name="description" value="Test XIS server to serve DispenseRequest (NL: VerstrekkingsVerzoek) - HPK geneesmiddel"/>
          <nts:with-parameter name="accept" value="xml"/>
-         <nts:with-parameter name="params"
-            value="?category=http://snomed.info/sct|52711000146108&amp;medication.code=urn:oid:2.16.840.1.113883.2.4.4.10|100153&amp;_include=MedicationRequest:medication"/>
+         <nts:with-parameter name="params" value="?category=http://snomed.info/sct|52711000146108&amp;_include=MedicationRequest:medication"/>
       </nts:include>
       <nts:include value="assert-responseSearchBundleSuccess" scope="common"/>
       <nts:include value="assert-responseBundleContent" scope="common"/>
-      <nts:include value="assert-returnCount" scope="project">
-         <nts:with-parameter name="description"
-                       value="Confirm that the returned searchset Bundle contains 1 DispenseRequest."/>
-         <nts:with-parameter name="expression"
-                       value="Bundle.entry.where(resource.is(MedicationRequest)).count() = 1"/>
+      <nts:include value="assert-returnCount-at-least-one-vv.xml" scope="project"/> 
```

Comment:
Ah maar als hier dan blijkt dat je een ...-at-least-one-ma.xml, .. -vv.xml en misschien nog wel een ander hebt, ben ik er misschien toch wel voor om de oplossing in m'n andere comment te gebruiken om het aantal verschillende includes wat te verminderen.

Reply:
Wat is er op tegen om verschillende includes te gebruiken? Ik bel je hier seperaat wel even over :)